src/main/java/dk/kb/storage/facade/DsStorageFacade.java

```diff
@@ -331,7 +377,17 @@ private static void validateIdHasOriginPrefix(String origin, String recordId) {
         }
     }
     
-
+    /**
+     * RecordType is not null
+     * @param origin name.
```

Comment:
There is no param called origin in the method signature, please remove this from the javadoc

Reply:
fixed